### PR TITLE
Pol 390 add name field to escalation block 

### PR DESCRIPTION
**Pull request by KVivek03**
Pol 390 add name field to escalation block 
Updated policies escalation label to maintain consistency

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: operational/dbaas/aws/rds_backup/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Changelog
+# Changelog
+
+## v2.3
+
```
[the remark this answers is not in the document]
Comment (KVivek03) [reply operational/dbaas/aws/rds_backup/CHANGELOG.md:4]: resolved

File: cost/aws/old_snapshots/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.0
```
Comment (nshivappa) [inline cost/aws/old_snapshots/CHANGELOG.md:3]: The version should be 2.10

File: cost/aws/reserved_instances/expiration/expired_ris.pt [hunk 3]
```diff
@@ -7,7 +7,7 @@ severity "medium"
 category "Cost"
 tenancy "single"
 info(
-  version: "1.9",
+  version: "2.0",
```
Comment (nshivappa) [inline cost/aws/reserved_instances/expiration/expired_ris.pt:10]: The version should be 1.10

File: cost/aws/reserved_instances/expiration/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0
```
Comment (nshivappa) [inline cost/aws/reserved_instances/expiration/CHANGELOG.md:3]: The version should be 1.10

File: cost/aws/old_snapshots/aws_delete_old_snapshots.pt [hunk 5]
```diff
@@ -5,7 +5,7 @@ short_description "Checks for snapshots older than specified number of days and,
 category "Cost"
 severity "low"
 info(
-  version: "2.9",
+  version: "3.0",
```
Comment (nshivappa) [inline cost/aws/old_snapshots/aws_delete_old_snapshots.pt:8]: The version should be 2.10

File: cost/aws/unused_ip_addresses/aws_unused_ip_addresses.pt [hunk 6]
```diff
@@ -409,8 +409,8 @@ end
```
Comment (nshivappa) [inline cost/aws/unused_ip_addresses/aws_unused_ip_addresses.pt:409]: The version is not updated for this file.

File: cost/schedule_instances/CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0
```
Comment (nshivappa) [inline cost/schedule_instances/CHANGELOG.md:3]: The version should be 1.10

File: cost/schedule_instances/schedule_instances.pt [hunk 8]
```diff
@@ -6,7 +6,7 @@ long_description ""
 severity "medium"
 category "Cost"
 info(
-  version: "1.9",
+  version: "2.0",
```
Comment (nshivappa) [inline cost/schedule_instances/schedule_instances.pt:9]: The version should be 1.10

File: msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt [hunk 9]
```diff
@@ -11,7 +11,7 @@ _Note 1: The last 3 days of data in the current week or month will contain incom
 _Note 2: The account you apply the policy to is unimportant as Optima metrics are scoped to the Org._  
 _Note 3: Exchange rates are calculated at execution time using [https://exchangeratesapi.io/](https://exchangeratesapi.io/)._  
 See [README](https://github.com/rightscale/policy_templates/tree/master/msp/cost/master_org_cost_policy_currency) for more details"
-long_description "Version: 1.0"
+long_description "Version: 1.1"
```
Comment (nshivappa) [inline msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt:14]: Can we add an info block and mention the version and other fields in info block

File: operational/compute_instance_migration/risc-compute-instance-migration-recos.pt [hunk 10]
```diff
@@ -321,5 +321,8 @@ EOS
 end
```
Comment (nshivappa) [inline operational/compute_instance_migration/risc-compute-instance-migration-recos.pt:322]: Is this policy supports new UI, as this policy is constructing the incident table dynamically in detail_template.

File: security/security_groups/rules_without_descriptions/CHANGELOG.md [hunk 11]
```diff
@@ -1,42 +1,46 @@
...
-## v1.1
-
-- Updating email from string to list
-
-## v1.0
-
-- Initial Release
+# Changelog
+
+## v2.0
```
Comment (nshivappa) [inline security/security_groups/rules_without_descriptions/CHANGELOG.md:3]: The version should be 1.10

File: security/security_groups/rules_without_descriptions/security_group_rules_without_descriptions.pt [hunk 12]
```diff
@@ -6,7 +6,7 @@ long_description ""
 severity "medium"
 category "Security"
 info(
-  version: "1.9",
+  version: "2.0",
```
Comment (nshivappa) [inline security/security_groups/rules_without_descriptions/security_group_rules_without_descriptions.pt:9]: The version should be 1.10

File: cost/aws/reserved_instances/compute_purchase_recommendation/CHANGELOG.md [hunk 13]
```diff
@@ -1,4 +1,9 @@
 # Changelog
 
+## v0.1
```
Comment (nshivappa) [inline cost/aws/reserved_instances/compute_purchase_recommendation/CHANGELOG.md:3]: Can we change versions to 1.0 and 1.1

File: cost/aws/reserved_instances/compute_purchase_recommendation/aws_reserved_instance_recommendations_with_purchase.pt [hunk 14]
```diff
@@ -2,7 +2,7 @@ name "AWS Reserved Instances Recommendations with Purchase"
 rs_pt_ver 20180301
 type "policy"
 short_description "A policy that sends email notifications when AWS RI Recommendations are identified and allows the user the make RI purchases after approval. NOTE: These RI Purchase Recommendations are generated by AWS. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/reserved_instances/recommendations) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 0.0"
+long_description "Version: 0.1"
```
Comment (nshivappa) [inline cost/aws/reserved_instances/compute_purchase_recommendation/aws_reserved_instance_recommendations_with_purchase.pt:5]: Can we add info block and mention version and other fields inside info block.

File: cost/aws/reserved_instances/compute_purchase_recommendation/aws_reserved_instance_recommendations_with_purchase.pt [hunk 15]
```diff
@@ -289,7 +289,9 @@ EOS
 end
```
Comment (nshivappa) [inline cost/aws/reserved_instances/compute_purchase_recommendation/aws_reserved_instance_recommendations_with_purchase.pt:290]: Please add incident resource table.

**Review by nshivappa — DISMISSED** [review]
LGTM

File: cost/low_service_usage/low_service_usage.pt [hunk 16]
```diff
@@ -293,7 +293,7 @@ end
 escalation "esc_budget_alert" do
   automatic true
   label "Send Email"
-  description "Send incident email"
+  description "Send Incident email"
```
Comment (nshivappa) [inline cost/low_service_usage/low_service_usage.pt]: Can we revert it to the "incident" as it is in lower case in all other policies?

**Review by nshivappa — APPROVED** [review]

**Pull request merged**